### ISSUE TITLE
Update `run_model.sh` script to specify all model parameters

### ISSUE THREAD

**Issue by dbeinhauer** [dbeinhauer/mcs-source#22]
Currently the script `run_model.sh` which is used to train model in the background process on CGG machines is outdated and it is possible to specify only learning rate, number of epochs and model type. We want to update it to specify ideally all possible model arguments.

**Comment by dbeinhauer** [comment]
Will not be done since we transfered to Metacentrum and experiments thus need not to be run there.
